src/components/pages/homepage/datasets.js

```diff
@@ -9,15 +9,15 @@ export default () => {
       whiteHouse: file(relativePath: { regex: "/white-house.png/" }) {
         relativePath
         childImageSharp {
-          fluid(maxWidth: 1200, traceSVG: { color: "#A7DEF6" }) {
+          fluid(maxWidth: 1200, traceSVG: { color: "#182E4E" }) {
             ...GatsbyImageSharpFluid_tracedSVG
           }
         }
       }
       covidExit: file(relativePath: { regex: "/covid-exit-strategy.png/" }) {
         relativePath
         childImageSharp {
-          fluid(maxWidth: 1200, traceSVG: { color: "#A7DEF6" }) {
+          fluid(maxWidth: 1200, traceSVG: { color: "#F7CB82" }) {
```

Comment:
Can we make this a dark color, like maybe sample the black from the `State's progress...` headline?

Right now, this renders in orange, which is a little distracting.